### PR TITLE
chore: update workflows from templates

### PR DESCRIPTION
**Pull request by nextcloud-command**
This PR was automatically generated by the workflow dispatcher
Please check that there are no reverted changes before merging ⚠

See templates in https://github.com/nextcloud/.github

### CODE REVIEW

File: .github/workflows/phpunit-sqlite.yml [hunk 1]
```diff
@@ -123,15 +132,6 @@ jobs:
         working-directory: apps/${{ env.APP_NAME }}
         run: composer run test:unit
 
-      - name: Upload Unit coverage
```
Comment (susnux) [inline .github/workflows/phpunit-sqlite.yml:126]: should stay

File: .github/workflows/phpunit-sqlite.yml [hunk 2]
```diff
@@ -150,15 +150,6 @@ jobs:
         working-directory: apps/${{ env.APP_NAME }}
         run: composer run test:integration
 
-      - name: Upload Integration coverage
```
Comment (susnux) [inline .github/workflows/phpunit-sqlite.yml:153]: should also stay

File: .github/workflows/psalm.yml [hunk 3]
```diff
@@ -12,34 +12,10 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  changes:
```
Comment (susnux) [inline .github/workflows/psalm.yml:15]: should stay

File: .github/workflows/phpunit-sqlite.yml (hunk 1 above)
Comment (Chartman123) [reply .github/workflows/phpunit-sqlite.yml]: added back :)

File: .github/workflows/psalm.yml (hunk 3 above)
Comment (Chartman123) [reply .github/workflows/psalm.yml]: added back :)

Comment (Chartman123) [reply .github/workflows/psalm.yml]: The other changes further down, too?

File: .github/workflows/phpunit-sqlite.yml (hunk 2 above)
Comment (Chartman123) [reply .github/workflows/phpunit-sqlite.yml]: added back :)

File: .github/workflows/psalm.yml (hunk 3 above)
Comment (susnux) [reply .github/workflows/psalm.yml]: Yes

**Review by Chartman123 — APPROVED** [review]

**Pull request merged**